py3status/modules/air_quality.py

```diff
@@ -150,9 +149,7 @@ def post_config_hook(self):
 
     def _get_aqi_data(self):
         try:
-            return self.py3.request(
-                self.url, params=self.auth_token, timeout=self.request_timeout
-            ).json()
+            return self.py3.request(self.url, params=self.auth_token).json()
```

Comment:
This is not hiding, if I understand correctly after this PR it will no longer be possible to customize `request_timeout` on a per-module basis. Correct? If so, why is this preferred?

Reply:
This is hardcoded in `post_config_hook` therefore it is not customizable on `air_quality` right now and still won't be after this PR. For those with already customizable config, I hide them and use `getattr()` just in case some user actually do use `request_timeout`.

The only useful module I could see for keeping `request_timeout` is maybe `whatismyip`... unless you think users don't care much about extra 5 seconds delay.

Reply:
I would rather mark timeout params as deprecated and move to consistent request_timeout

Then I think that we should make py3.request honor any request_timeout parameter set on the module which would allow us to not getattr the attribute

I have this ready to be pushed, if you're okay I'll add a commit to the current PR so you can see how it looks

Reply:
Yeah! Sounds okay! Fire away.

Reply:
done, the only leftover is pingdom which do NOT use the py3.request helper, maybe it's time to migrate it so we have everything consistent?

Reply:
Done. Looks good. :+1: 
```
2019-01-16 07:08:22 INFO Module `exchange_rate`: START TIMEOUT: None
2019-01-16 07:08:22 INFO Module `exchange_rate`: END TIMEOUT: 10
2019-01-16 07:09:23 INFO Module `whatismyip`: START TIMEOUT: None
2019-01-16 07:09:23 INFO Module `whatismyip`: END TIMEOUT: 10
```

Reply:
Now we only need to update the py3 helper documentation because we just made `request_timeout` a generic parameter for all URL request based modules please

Reply:
I allowed myself to rephrase the documentation a bit